test/e2e/PISP_3PR_Transfer.test.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-named-as-default-member */
```

Comment:
It's amazing how so much work comes down to just 73 lines of e2e test...

Reply:
we don't want PISP to write a lot of code :-P